<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Devices</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="1" >
         <widget class="QPushButton" name="pushButtonScanCan" >
          <property name="text" >
           <string>Scan CAN</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QTreeView" name="treeView" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Packets</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTableView" name="tableViewPackets" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Options</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="1" >
         <widget class="QLabel" name="statusLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>4</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QPushButton" name="pushButtonConnect" >
          <property name="maximumSize" >
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2" >
         <widget class="QLineEdit" name="lineEditHost" >
          <property name="text" >
           <string>rl</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="lineEditPort" >
          <property name="maximumSize" >
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>2342</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
